### PR TITLE
Create trouble-postgres.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Create documentation for https://github.com/mattermost/mattermost-server/issues/22249

### CODE REVIEW

**Review by cwarnermm — APPROVED** [review]
Great PR, @justinegeffen! Well done!

File: source/install/trouble-postgres.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+PostgreSQL installation troubleshooting
+=======================================
+
+Mattermost supports PostgreSQL v10+.
+
+PostgreSQL v15 introduces changes that may affect compatibility with previous releases. In order to ensure compatability, take note of the following recommendations:
+
+- Remove the ``PUBLIC`` creation permission on the public schema.
+- For existing databases, especially those having multiple users, consider revoking ``CREATE`` permission on the public schema to adopt this new default.
+- For new databases which have no need to defend against insider threats, granting ``CREATE`` permission will yield the behavior of prior releases.
```
Comment (agnivade) [inline source/install/trouble-postgres.rst:10]: You have just copied verbatim from the postgres docs :) And a lot of it is incorrect for Mattermost. If we revoke the CREATE permission, Mattermost won't work.

IMO this is confusing to the system admins, and it does not mention the command that is needed to fix things.

I'd just say that if you are deploying a fresh installation of Postgres v15, run this command: `GRANT CREATE ON SCHEMA public TO PUBLIC;` to let Mattermost work. That's it.

Comment (cwarnermm) [reply source/install/trouble-postgres.rst:10]: Thanks, @agnivade. I created these notes from the details Engineering shared in the discussion around these clarifications. Really appreciate you providing the abridged version :)

Comment (justinegeffen) [reply source/install/trouble-postgres.rst:10]: Thanks, @agnivade! I lifted this off the linked ticket thread. Thanks for clarifying!

**Pull request merged**